<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClient</class>
 <widget class="QDialog" name="AddClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>590</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>501</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Age:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ageBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Gender:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="genderBox">
        <item>
         <property name="text">
          <string>Not chose</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Mariatal status:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="marriageBox">
        <item>
         <property name="text">
          <string>Unknown</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Single</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Divorced</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Married</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Educational Level:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="edBox">
        <item>
         <property name="text">
          <string>Graduate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Post-Graduate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Doctorate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High School</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Uneducated</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>College</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Unknown</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Income:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="incomeBox">
        <item>
         <property name="text">
          <string>Less than $40K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>$40K - $60K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>$60K - $80K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>$80K - $120K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Unknown</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Card Category:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cardBox">
        <item>
         <property name="text">
          <string>Blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Silver</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Plantinum</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Total Revolving Balance:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="balanceLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Credit Limit:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="limitLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Utilization Ration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UtilLine"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add information about the new client</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
